threads-api/__test__/getTokenWithProxy.test.ts

```diff
@@ -25,4 +35,8 @@ describeIf(!!credentials)('getToken', () => {
       TIMEOUT,
     );
   });
+
+  it('should use the correct agent/proxy URL', () => {
+    expect(HttpsProxyAgentSpy).toHaveBeenCalledWith(proxy);
```

Comment:
Wait I think this test is still wrong, this have no effect since we're not calling `getToken` here